tasks/notify_error.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from dataclasses import dataclass
+
+from asgiref.sync import async_to_sync
+from sqlalchemy.orm import Session
+
+from celery_config import notify_error_task_name
+from database.enums import ReportType
+from database.models import Commit, CommitReport, Upload
+from helpers.checkpoint_logger import from_kwargs as checkpoints_from_kwargs
+from helpers.checkpoint_logger.flows import UploadFlow
+from helpers.notifier import BaseNotifier, NotifierResult
+from services.yaml import UserYaml
+from tasks.base import BaseCodecovTask, celery_app
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class ErrorNotifier(BaseNotifier):
+    failed_upload: int
+    total_upload: int
+
+    def build_message(
+        self,
+    ) -> str:
+        error_message = f"❗️ We couldn't process [{self.failed_upload}] out of [{self.total_upload}] uploads. Codecov cannot generate a coverage report with partially processed data. Please review the upload errors on the commit page."
```

Comment:
does this emoji reference work? When I changed notifiers with emojis before I had to use something like `:tada:` (see `_create_welcome_message`)